prime-router/src/main/kotlin/Metadata.kt

```diff
@@ -353,19 +352,6 @@ class Metadata : Logging {
      */
     internal var tablelastCheckedAt = Instant.now()
 
-    private fun loadLookupTables(filePath: String): Metadata {
-        val catalogDir = File(filePath)
-        if (!catalogDir.isDirectory) error("Expected ${catalogDir.absolutePath} to be a directory")
-        try {
-            readAllTables(catalogDir) { tableName: String, table: LookupTable ->
```

Comment:
`readAllTables` now isn't used anywhere and can be removed.

Reply:
Removed.  I had to force a reindex in my IntelliJ to see it.  Removed some others I found too.  Thx

> `readAllTables` now isn't used anywhere and can be removed.